lochness/box/__init__.py

```diff
@@ -47,6 +48,35 @@ def base(Lochness, module_name):
                    .get('base', '')
 
 
+def get_access_token(client_id: str, client_secret: str, user_id: str) -> str:
+    '''Get new access token using Box API
+
+    Key Argument:
+        client_id: Client ID from the box app from box dev console, str
+        client_secret: Client secret from the box app from box dev console, str
+        user_id: user id from the box app from box dev console, str of digits.
+
+    Returns:
+        api_token: API TOKEN used to pull data, str.
+
+    Notes:
+        Current version of get_access_token function uses User ID rather
+        than Enterprise ID. (Both works in pulling data from BOX account)
+    '''
+    url = "https://api.box.com/oauth2/token"
+    headers = {"Content-Type": "application/x-www-form-urlencoded"}
+    data = {"client_id": client_id,
+            "client_secret": client_secret,
+            "grant_type": "client_credentials",
+            "box_subject_type": "user",
+            "box_subject_id": user_id}
```

Comment:
And when you query the `url` with this `data`, does it provide you a new `api_token`?

It is unorthodox for box to require all three of `client_id`, `client_secret`, and `box_subject_id` to authenticate though!

Reply:
>And when you query the url with this data, does it provide you a new api_token?

Yes, it does.